### PR TITLE
Added flag and liquidation position functions

### PR DESCRIPTION
**Pull request by TossAyeCoin**

### CODE REVIEW

File: module/kwenta/kwenta.py [hunk 1]
```diff
@@ -280,12 +280,94 @@ def can_liquidate(self, token_symbol: str) -> dict:
...
         liquidation_check = market_contract.functions.canLiquidate(
-            self.wallet_address).call()
+            wallet_address).call()
         liquidation_price = market_contract.functions.liquidationPrice(
-            self.wallet_address).call()
+            wallet_address).call()
         return {"liq_possible": liquidation_check,
                 "liq_price": liquidation_price}
 
+    def liquidatePosition(self, token_symbol: str, wallet_address: str, execute_now: bool = False) -> dict:
```
Comment (Tburm) [inline module/kwenta/kwenta.py:289]:
```suggestion
    def liquidate_position(self, token_symbol: str, wallet_address: str, execute_now: bool = False) -> dict:
```

File: module/kwenta/kwenta.py [hunk 2]
```diff
@@ -280,12 +280,94 @@ def can_liquidate(self, token_symbol: str) -> dict:
...
+            else:
+                return {
+                    "token": token_symbol.upper(),
+                    "tx_data": tx_params}
+        else:
+            return {
+                "token": token_symbol.upper(),
+                "tx_data": "N/A, Cannot Liquidate Position."}
+
+    def flagPosition(self, token_symbol: str, wallet_address: str, execute_now: bool = False) -> dict:
```
Comment (Tburm) [inline module/kwenta/kwenta.py:330]:
```suggestion
    def flag_position(self, token_symbol: str, wallet_address: str, execute_now: bool = False) -> dict:
```

File: module/kwenta/kwenta.py [hunk 3]
```diff
@@ -280,12 +280,94 @@ def can_liquidate(self, token_symbol: str) -> dict:
...
+        Returns
+        ----------
+        Dict: flag Liquidation of position
+        """
+        market_contract = self.get_market_contract(token_symbol)
+        liquidation_check = market_contract.functions.canLiquidate(
+            self.wallet_address).call()
+        # check for if liquidation is possible
+        if liquidation_check == True:
+            market_contract.functions.flagPosition(wallet_address).call()
```
Comment (Tburm) [inline module/kwenta/kwenta.py:350]: What does this `.call()` do if we're executing the transaction after this?

File: module/kwenta/kwenta.py [hunk 4]
```diff
@@ -280,12 +280,94 @@ def can_liquidate(self, token_symbol: str) -> dict:
...
+        Returns
+        ----------
+        Dict: Liquidation of position
+        """
+        market_contract = self.get_market_contract(token_symbol)
+        liquidation_check = market_contract.functions.canLiquidate(
+            self.wallet_address).call()
+        # check for if liquidation is possible
+        if liquidation_check == True:
+            market_contract.functions.liquidatePosition(wallet_address).call()
```
Comment (Tburm) [inline module/kwenta/kwenta.py:309]: Same as `flag_position`, what does this `.call()` do if we're encoding parameters and executing using the raw transaction?

**Review by Tburm — COMMENTED** [review]
Left naming comments where we should follow the snake case convention started earlier.

File: module/kwenta/kwenta.py (hunk 3 above)
Comment (TossAyeCoin) [reply module/kwenta/kwenta.py]: Yep these are both extra. Removed these.

**Pull request merged**